central/src/main/webapp/components/ModelNavigator.jsx

```diff
@@ -86,14 +98,99 @@ export default function ModelNavigator(props) {
+                                        inputProps={{style: { backgroundColor:"white"},}}
+                                    />
+                                </TreeItem>
+                                <TreeItem nodeId="Version" label="Version">
+                                    <div onChange={modelVersionFilterOnChange}>
+                                        <button disabled={versionValue=="0.0.1"} value="0.0.1" onClick={modelVersionFilterOnChange} > 0.0.1 </button>
+                                        <button disabled={versionValue=="0.0.2"} value="0.0.2" onClick={modelVersionFilterOnChange} > 0.0.2 </button>
+                                        <button disabled={versionValue=="0.0.3"} value="0.0.3" onClick={modelVersionFilterOnChange} > 0.0.3 </button>
+                                        <button value="" onClick={modelVersionFilterOnChange} > Clear </button>
+                                    </div>
+                                </TreeItem>
+                                <TreeItem nodeId="Type" label="Type">
+                                    <div onChange={modelVersionFilterOnChange}>
+                                       <button disabled={typeValue=='CV.ACTION_RECOGNITION'} value= 'CV.ACTION_RECOGNITION' onClick={modelTypeFilterOnChange} > action_recognition </button>
+                                       <button disabled={typeValue=='CV.IMAGE_CLASSIFICATION'} value= 'CV.IMAGE_CLASSIFICATION' onClick={modelTypeFilterOnChange} > image_classification </button>
+                                       <button disabled={typeValue=='CV.INSTANCE_SEGMENTATION'} value= 'CV.INSTANCE_SEGMENTATION' onClick={modelTypeFilterOnChange} > instance_segmentation </button>
+                                       <button disabled={typeValue=='CV.OBJECT_DETECTION'} value= 'CV.OBJECT_DETECTION' onClick={modelTypeFilterOnChange} > object_detection </button>
+                                       <button disabled={typeValue=='CV.POSE_ESTIMATION'} value='CV.POSE_ESTIMATION' onClick={modelTypeFilterOnChange} > pose_estimation </button>
+                                       <button disabled={typeValue=='NLP.QUESTION_ANSWER'} value='NLP.QUESTION_ANSWER' onClick={modelTypeFilterOnChange} > question_answer </button>
+                                       <button disabled={typeValue=='NLP.SENTIMENT_ANALYSIS'} value='NLP.SENTIMENT_ANALYSIS' onClick={modelTypeFilterOnChange} > sentiment_analysis </button>
+                                       <button disabled={typeValue=='NLP.WORD_EMBEDDING'} value='NLP.WORD_EMBEDDING' onClick={modelTypeFilterOnChange} > word_embedding </button>
+                                       <button value='' onClick={modelTypeFilterOnChange} > Clear </button>
+                                    </div>
+                                </TreeItem>
+                            </TreeView>
+                            <TreeItem nodeId="Models" label="Models">
```

Comment:
Rather than building a completely new UI, think about if it can be merged with an existing UI. Can you show models using the existing model list?

Reply:
<img width="883" alt="Screen Shot 2021-04-01 at 1 45 59 PM" src="https://user-images.githubusercontent.com/65963140/113352053-a00dd700-92f0-11eb-851c-3b7d46f3cd45.png">

Reply:
This is the new look I came up with for the Model Navigation system. I think it looks a little cleaner than the previous. Let me know if there's any modification that you think would make it look better.